### DISCUSSION TITLE
Separed theme for content and UI

### DISCUSSION THREAD

**Discussion by Plx0wn** [standardnotes/forum#2902] [Suggestions]
Hi there !

There is the situation :

- I use a lot **rich text** and **spreadsheet**
- I love the **dark theme** (look better and I work a lot the night)
- I better see **black letters and numbers on white background**.

It is possible to add in our preference a separated theme for the main UI and the content ?

For example : 
![example](https://user-images.githubusercontent.com/18502986/203124470-7485da40-a379-4a88-aa84-01b77c352960.jpg)

Thanks for the reading !

**Comment by effieeee** [comment]
Thank you as well for your suggestion, though there isn't anything planned yet relating to this sort of customization on the themes, so I'm afraid this would need you to build a custom theme or plugin for now 😞
